lib/discourse_automation/event_handlers.rb

```diff
@@ -65,6 +65,59 @@ def self.handle_post_created_edited(post, action)
         end
     end
 
+    def self.handle_user_updated(user)
+      return if user.id < 0
+
+      name = DiscourseAutomation::Triggerable::USER_UPDATED
+
+      DiscourseAutomation::Automation
+        .where(trigger: name, enabled: true)
+        .find_each do |automation|
+          if automation.trigger_field("first_post_only")["value"] &&
+               UserCustomField.find_by(name: automation.trigger_field("automation_name")["value"])
+            next
+          end
```

Comment:
It's not very clear to me what it does, maybe give a variable name to the find_by so it can be more explicit what this db query is checking